src/errors/test-run/templates.js

```diff
@@ -218,8 +215,14 @@ export default {
     `),
 
     [TYPE.regeneratorInFunctionArgumentOfHybridFunctionError]: err => markup(err, `
-        Hybrid function argument is a function that contains either generators or the <code>async/await</code> syntax. These features cannot be used in client code. Use Promises instead.
+        Hybrid function argument is a function that contains either generators or the <code>async/await</code> syntax. These features cannot be used in hybrid function code. Use Promises instead.
 
         ${err.getCallsiteMarkup()}
+    `),
+
+    [TYPE.domNodeHybridResultError]: err => markup(err, `
+       Regular Hybrid functions cannot return DOM elements. Use <code>Selector</code> functions for this purpose.
```

Comment:
Client functions cannot return DOM elements. Use <code>Selector</code> functions for this purpose.